man/docker-logs.1.md

```diff
@@ -42,11 +42,17 @@ logging drivers.
 **--tail**="all"
    Output the specified number of lines at the end of logs (defaults to all logs)
 
-The `--since` option shows only the container logs generated after
-a given date. You can specify the date as an RFC 3339 date, a UNIX
-timestamp, or a Go duration string (e.g. `1m30s`, `3h`). Docker computes
-the date relative to the client machine’s time. You can combine
-the `--since` option with either or both of the `--follow` or `--tail` options.
+The `--since` option can be Unix timestamps, date formated timestamps, or Go 
+duration strings (e.g. `10m`, `1h30m`) computed relative to the client machine’s 
+time. Supported formats for date formated time stamps include RFC3339Nano, 
+RFC3339, `2006-01-02T15:04:05`, `2006-01-02T15:04:05.999999999`, 
+`2006-01-02Z07:00`, and `2006-01-02`. The local timezone on the client will be 
+used if you do not provide either a `Z` or a `+-00:00` timezone offset at the 
+end of the timestamp.  When providing Unix timestamps enter 
+seconds[.nanoseconds], where seconds is the number of seconds that have elapsed 
```

Comment:
Since the code performs a shift (i.e. 123.456 is four-fifty-six milliseconds, rather than four-fifty-six nanoseconds), should this not say `seconds[.subseconds]`?